input.go

```diff
@@ -289,3 +291,31 @@ func (ins Inputs) Less(i, j int) bool {
 func (ins Inputs) Swap(i, j int) {
 	ins[i], ins[j] = ins[j], ins[i]
 }
+
+// CreateVM ...
+func (ins Inputs) CreateVM(srcdir string) (*otto.Otto, error) {
```

Comment:
Because this method belongs to `Inputs` struct, I prefer a name `ToJavaScriptVM` instead of `Create...`